tests/Instances.hs

```diff
@@ -145,3 +149,12 @@ instance Arbitrary Version where
 makeVersion :: [Int] -> Version
 makeVersion b = Version b []
 #endif
+
+instance Arbitrary (Proxy a) where
+    arbitrary = pure Proxy
+
+instance Arbitrary b => Arbitrary (Tagged a b) where
+    arbitrary = Tagged <$> arbitrary
+
+instance Arbitrary a => Arbitrary (Const a b) where
+    arbitrary = Const <$> arbitrary
```

Comment:
Not yet. I'll make a PR soonish. Not sure whether Nick would like to depend on tagged though.